Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -253,20 +253,21 @@ def remove_irrelevant_incident_ids(last_run_fetched_ids: dict[str, dict[str, str
         extensive_log(f'[SplunkPy] Checking if {incident_id} is relevant to fetch window')
         if isinstance(incident_occurred_time, dict):
             # To handle last fetched IDs
-            # Last fetched IDs hold the occurred time that they were seen, and will be deleted from
-            # the last fetched IDs once they pass the fetch window
-            incident_start_datetime = datetime.strptime(incident_occurred_time.get('occurred_time', ''), SPLUNK_TIME_FORMAT)
-            if incident_start_datetime >= window_start_datetime:
+            # Last fetched IDs hold the occurred time that they were seen, which is basically the end time of the fetch window
+            # they were fetched in, and will be deleted from the last fetched IDs once they pass the fetch window
+            incident_occurred_datetime = datetime.strptime(incident_occurred_time.get('occurred_time', ''), SPLUNK_TIME_FORMAT)
```

Comment:
I think the name `incident_occurred_datetime` it's confusing 
as we actually try to say the end of the incident's fetch window

Reply:
I changed it to **incident_window_end_datetime**, wdyt?